### ISSUE TITLE
Memory leaks with aiVector3D

### ISSUE THREAD

**Issue by AnnoyingB** [assimp/assimp#4982]
```c++
if (aimesh->HasPositions()) {
	const aiVector3D* aPos = &(aimesh->mVertices[j]);
	vertex.x = aPos->x;
	vertex.y = aPos->y;
	vertex.z = aPos->z;
}
```

It randomly leaks memory, sometimes it is for y, sometimes z but never one on x. I searched up tutorials, copied them and they still caused me memory leaks.

OS: Windows 11 64bit
Visual Studio 2022 v143
ISO C++ 14 standard

Edit: I set the `mVertices[j]` to `mVertices[0]` and it doesn't give an error but just freezes (probably a problem with my other code).

**Comment by turol** [comment]
`aiVector3D` is a simple type with no pointers. It can't leak memory. Something else is leaking vectors.

How are you checking for memory leaks? What loader are you using? Can you share your model? Can you create a simple self-contained example which demonstrates the problem.

**Comment by AnnoyingB** [comment]
> How are you checking for memory leaks?

I am using vs2022 and when I check what aiVector3D has inside of it, it said `{ x: ???, y: ??? z: ??? }`. I found out this is the way I was storing it as a pointer but when I run my code, all the values are 0 and in the example it gives different values. It now just gives a access violation.

> What loader are you using?

I am just using `Importer.ReadFile();`.

> Can you share your model?

It is just a scaled down blender cube. I deleted all lighting, cameras etc. so there is only the cube model.

> Can you create a simple self-contained example which demonstrates the problem.

[AssimpTest.zip](https://github.com/assimp/assimp/files/10852858/AssimpTest.zip)
